nginx.conf

```diff
@@ -1,27 +1,87 @@
-# Experimental nginx configuration file for domain and subdomain Known installations
-# Originally written by Kyle Mahan
+user www-data;
 
-server {
+events {
+  worker_connections 2048;
+  multi_accept       on;
+  use                epoll;
+}
+
+http {
+  sendfile           on;
+  tcp_nopush         on;
+  tcp_nodelay        on;
+  keepalive_timeout  15;
+
+  upstream backend {
+    server localhost:9000; #or your php-fpm socket
+  }
+
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
 
+  gzip on;
+  gzip_vary on;
+  gzip_min_length 10240;
+  gzip_proxied expired no-cache no-store private auth;
+  gzip_types text/plain text/css text/xml application/javascript application/x-javascript application/xml;
+  gzip_disable "MSIE [1-6]\.";
+
+  server {
     listen 80;
-    root /home/vagrant/idno;
+    client_max_body_size 20M;
+    
+    root /var/www/html;
     index index.php;
-    server_name vagrant.dev;
 
-    location ~ (\.ini|known.php)$ {
-        deny all;
+    open_file_cache          max=2000 inactive=20s;
+    open_file_cache_valid    60s;
+    open_file_cache_min_uses 5;
+    open_file_cache_errors   off;
+    
+    location ~* \.(css|js|jpg|jpeg|gif|png|ico|xml)$ {
+      expires           30d;
+      add_header Pragma public;
+      add_header Cache-Control "public";
+      rewrite ^/css/[0-9]+/(.*) /css/$1 last;
+      rewrite ^/js/[0-9]+/(.*) /js/$1 last;
+      try_files $uri $uri/ /index.php?$query_string;
     }
 
     location / {
-        try_files $uri $uri/ /index.php?$query_string;
+      try_files $uri $uri/ /index.php?$query_string;
     }
 
     location ~ \.php$ {
-        try_files $uri =404;
-        include /etc/nginx/fastcgi_params;
-        fastcgi_pass unix:/var/run/php5-fpm.sock;
-        fastcgi_index index.php;
-        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+      fastcgi_param  GATEWAY_INTERFACE  CGI/1.1;
```

Comment:
@paulcmal suggested `try_files $uri =404;` here: https://github.com/idno/Known/issues/1031#issuecomment-135958676

Worth keeping it?